src/v2i-hub/CDASimAdapter/test/TestCARMASimulationConnection.cpp

```diff
@@ -21,15 +22,16 @@ namespace CDASimAdapter {
     class TestCARMASimulationConnection : public ::testing::Test {
         protected:
             void SetUp() override {
-                // Initialize CARMA Simulation connection with (0,0,0) location.
+                // Initialize CARMA Simulation connection with (0,0,0) location and mock kafka producer.
```

Comment:
small issue but there is no mock kafka producer here.

Reply:
removed